.github/workflows/pr-closed.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         uses: actions/checkout@v3
       - run: echo "The ${{ github.repository }} repository has been cloned to the runner."
       - name: Configure AWS credentials from Dev account
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@v1-node16
```

Comment:
Out of interest why we're specifying node16 now?

Reply:
Its a weird fix, at the moment set output changes (doing it the new way) does not always actually work. Until github fix this issue the recommendation was to change the package, here is where i found this suggestion :) https://github.com/aws-actions/configure-aws-credentials/issues/521

Reply:
ah strange, all looks good though